tests/planner/ompl/OMPLTestHelpers.hpp

```diff
@@ -1,6 +1,8 @@
 #ifndef AIKIDO_TESTS_OMPL_OMPLTESTHELPERS_HPP_
 #define AIKIDO_TESTS_OMPL_OMPLTESTHELPERS_HPP_
 
+#include <math.h>
```

Comment:
Got it! Removed it for now though. Don't need it. Using `dmetric` for trajectory length calculation now as suggested previously.